tests/foreman/ui/test_host.py

```diff
@@ -1928,7 +1928,7 @@ def test_positive_gce_provision_end_to_end(
             # 1.1 UI based Assertions
             host_info = session.host.get_details(hostname)
             assert session.host.search(hostname)[0]['Name'] == hostname
-            assert host_info['properties']['properties_table']['Build'] == 'Installed'
+            assert host_info['properties']['properties_table']['Build'] == 'Installed clear'
```

Comment:
Whats `clear` here ? That should not appear and that doesn't have any meaning! 

Worth raising a bug ?

Reply:
it's not a bug, it a feature :D - it's a button to forget or erase that status line from the host info table -> /forget_status?status=1